src/plots/cartesian/axes.js

```diff
@@ -1137,7 +1133,7 @@ axes.tickText = function(ax, x, hover) {
 };
 
 function tickTextObj(ax, x, text) {
-    var tf = ax.tickfont || ax._gd._fullLayout.font;
+    var tf = ax.tickfont || {};
```

Comment:
After investigation, I'll keep the `|| {}`, if ok.

Whenever `ax.showticklabels: false`, [`ax.tickfont`](https://github.com/plotly/plotly.js/blob/954da8a806f814798cf6cbffa30087f9ca1d43d2/src/plots/cartesian/tick_label_defaults.js#L28-L38) isn't coerced and isn't defined here.

We could make sure that `tickTextObj` isn't called when `ax.showticklabels: false`, but this requires adding a few early returns in axes.js, 3d and gl2d convert routines which I believe isn't worth it.